docs/reference-use-cases.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Cluster API Reference Use Cases
+authors:
+  - "@vincepri"
+  - "@ncdc"
+  - "@detiber"
+reviewers:
+  - "@timothysc"
+  - "@justinsb"
+creation-date: 2019-04-08
+last-updated: 2019-04-08
+---
+
+# Cluster API Reference Use Cases
+
+This is a living document that is refined over time. It serves as a reference for use cases collected during post-v1alpha1 project redesign.
```

Comment:
There should also be some text here indicating that this doc is a staging area and not the canonical documentation for cluster-api use cases.